### ISSUE TITLE
[.Net][Issue]: AutogenDotnetInteractive doesn't restore dotnet-interactive service when system language is Chinese

### ISSUE THREAD

**Issue by feynmanloo** [microsoft/autogen#2660]
### Describe the issue

![image](https://github.com/microsoft/autogen/assets/20107191/1672d30a-0f71-4535-9229-d106f35c7c2b)
![image](https://github.com/microsoft/autogen/assets/20107191/7f905602-f37d-4368-9a8b-f3621fddf95f)

**Comment by LittleLittleCloud** [comment]
I haven’t tested dotnet interactive with asp net yet, would you mind sharing the code somewhere and I can debug into it

from the log the error is caused because dotnet interactive is not restored correctly. Note that the autogen.dotnetinteractive always try installing a specific version of dotnet interactive even though it’s installed globally. You can cd to work directory and see if you can open dotnet interactive cli there

**Comment by feynmanloo** [comment]
> I haven’t tested dotnet interactive with asp net yet, would you mind sharing the code somewhere and I can debug into it
> 
> from the log the error is caused because dotnet interactive is not restored correctly. Note that the autogen.dotnetinteractive always try installing a specific version of dotnet interactive even though it’s installed globally. You can cd to work directory and see if you can open dotnet interactive cli there

@LittleLittleCloud  thx a lot !
https://github.com/feynmanloo/AutogenDotnetInteractiveDemo

**Comment by LittleLittleCloud** [comment]
@feynmanloo Your code works on my local environment. May I ask which dotnet SDK do you have on your environment?

Here's my SDKs
```
dotnet --list-sdks
7.0.408 [C:\Program Files\dotnet\sdk]
8.0.204 [C:\Program Files\dotnet\sdk]
```

**Comment by feynmanloo** [comment]
> @feynmanloo Your code works on my local environment. May I ask which dotnet SDK do you have on your environment?
> 
> Here's my SDKs
> 
> ```
> dotnet --list-sdks
> 7.0.408 [C:\Program Files\dotnet\sdk]
> 8.0.204 [C:\Program Files\dotnet\sdk]
> ```
@LittleLittleCloud 
dotnet --list-sdks
6.0.420 [C:\Program Files\dotnet\sdk]
7.0.408 [C:\Program Files\dotnet\sdk]
8.0.204 [C:\Program Files\dotnet\sdk]

**Comment by LittleLittleCloud** [comment]
@feynmanloo I think I figure it out. The `InteractiveService` restores `dotnet-interactive` when it receives an `CommandLineInvocationException` with error message that contains `cannot find a tool in the manifest file that has a command named 'dotnet-interactive'`. But since your system language is Chinese, so the error message will also be in Chinese and it doesn't contain that specific message piece.

We will push a fix for this issue in 0.0.14. To unblock yourself, you can mannually restore the dotnet interactive by calling `RestoreDotnetInteractive`. This method is private though so you might need to use reflection to call that method. See this [link](https://stackoverflow.com/questions/135443/how-do-i-use-reflection-to-invoke-a-private-method) on how to do that

**Comment by feynmanloo** [comment]
How soon is this release expected?

**Comment by LittleLittleCloud** [comment]
@feynmanloo Might be 2-3 weeks, depends on the progress of other working items.

You can also consume the nightly build as well, which will be available once #2676 get merged into main

**Comment by feynmanloo** [comment]
> @feynmanloo Might be 2-3 weeks, depends on the progress of other working items.
> 
> You can also consume the nightly build as well, which will be available once #2676 get merged into main

@LittleLittleCloud  where can i get nightly build package ?

**Comment by feynmanloo** [comment]
> @feynmanloo Might be 2-3 weeks, depends on the progress of other working items.
> 
> You can also consume the nightly build as well, which will be available once #2676 get merged into main

@LittleLittleCloud 
can't find the newly nightly build package from myget, and i find this ci error:
https://github.com/microsoft/autogen/actions/runs/9073421528/job/24934786609

**Comment by LittleLittleCloud** [comment]
Looks like the API key is expired, I will update the key and rerun the workflow

**Comment by LittleLittleCloud** [comment]
@feynmanloo Updated
